### DISCUSSION TITLE
120Hz animations?

### DISCUSSION THREAD

**Discussion by thewirv** [software-mansion/react-native-reanimated#2472] [Ideas]
The last thing I've heard about Reanimated is that it "provides smooth 60 FPS animations". But are they locked to 60 FPS, or are they only limited by the refresh rate of the screen? Meaning, on 120Hz screens, that are becoming more and more popular (now even Apple is boarding that train with the 13 Pro), are Reanimated animations still locked to 60 FPS, or can they run in 120 FPS if the device is capable of it performance-wise?

Background: I myself am using Reanimated at work, and loving it so far. I have just been wondering about this recently.

**Comment by Sixzero** [comment]
I guess it is not locked at 60 FPS, usually we say 60 FPS because it runs on UI thread, which is really performant. 
I am not sure what part is native and what part is not, but I guess the animation part runs on native codes, so animations probably are 120 FPS.

**Comment by pistonsky** [comment]
It runs 120hz, I tested it. At least it feels like that (tested on Android).

**Comment by piaskowyk** [comment]
Yes, I can confirm that Reanimated supports 120 FPS on iOS and Android

**Comment by dondxniel** [comment]
Hello @piaskowyk do you mean it runs 120fps even on devices that don't support 120hz refresh rate?

**Reply by MatiPl01** [reply]
No, it adapts frame rate to the screen refresh frequency. You can see that in the performance monitor which can be enabled on a device or a simulator.
